client/blocks/reader-unsubscribed-feeds-search-list/reader-unsubscribed-feed-item.tsx

```
@@ -0,0 +1,108 @@
+import { Button } from '@automattic/components';
+import { Reader, SubscriptionManager } from '@automattic/data-stores';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { filterURLForDisplay } from '@wordpress/url';
+import { useTranslate } from 'i18n-calypso';
+import { SiteIcon } from 'calypso/blocks/site-icon';
+import ExternalLink from 'calypso/components/external-link';
+import { getFeedUrl, getSiteName, getSiteUrl } from 'calypso/reader/get-helpers';
+import { useDispatch } from 'calypso/state';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+
+type ReaderUnsubscribedFeedItemProps = {
+	feed: Reader.FeedItem;
+};
+
+const ReaderUnsubscribedFeedItem = ( { feed }: ReaderUnsubscribedFeedItemProps ) => {
+	if ( Number.isNaN( Number( feed.blog_ID ) ) ) {
+		throw new Error( 'Feed item blog_ID is NaN' );
```

Throwing an error like this would cause a WSOD. Is there a more graceful way of handling this? Failing silently and just don't show the item? Not a blocker for this PR.